app/_src/policies/meshcircuitbreaker.md

```diff
@@ -0,0 +1,303 @@
+```
+
+We will apply the configuration with `kubectl apply -f [..]`.
+{% endtab %}
+
+{% tab protocol Universal %}
+
+```yaml
+type: MeshCircuitBreaker
+name: backend-inbound-outlier-detection
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        outlierDetection:
+          interval: 5s
+          baseEjectionTime: 30s
+          maxEjectionPercent: 20
+          splitExternalAndLocalErrors: true
+          detectors:
+            totalFailures:
+              consecutive: 10
+            gatewayFailures:
+              consecutive: 10
+            localOriginFailures:
+              consecutive: 10
+            successRate:
+              minimumHosts: 5
+              requestVolume: 10
+              standardDeviationFactor: 1.9
+            failurePercentage:
+              requestVolume: 10
+              minimumHosts: 5
+              threshold: 85
+```
+
+We will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/{{ page.version }}/reference/http-api).
+{% endtab %}
+{% endtabs %}
+
+## Connection limits configuration
+
+- **`maxConnections`** - (optional) The maximum number of connections allowed to be made to the upstream cluster. If not
+  specified then equal to "1024".
+- **`maxConnectionPools`** - (optional) The maximum number of connection pools per cluster that are concurrently
+  supported at once. Set this for clusters which create a large number of connection pools. If not specified, the
+  default is unlimited.
+- **`maxPendingRequests`** - (optional) The maximum number of pending requests that are allowed to the upstream cluster.
+  This limit is applied as a connection limit for non-HTTP traffic. If not specified then equal to "1024".
+- **`maxRetries`** - (optional) The maximum number of parallel retries that will be allowed to the upstream cluster. If
+  not specified then equal to "3".
+- **`maxRequests`** - (optional) The maximum number of parallel requests that are allowed to be made to the upstream
+  cluster. This limit does not apply to non-HTTP traffic. If not specified then equal to "1024".
+
+## Outlier detection configuration
+
+- **`disabled`** - (optional) When set to true, outlierDetection configuration won't take any effect.
+- **`interval`** - (optional) The time interval between ejection analysis sweeps. This can result in both new ejections
+  and hosts being returned to service.
+- **`baseEjectionTime`** - (optional) The base time that a host is ejected for. The real time is equal to the base time
+  multiplied by the number of times the host has been ejected.
+- **`maxEjectionPercent`** - (optional) The maximum % of an upstream cluster that can be ejected due to outlier
+  detection. Defaults to 10% but will eject at least one host regardless of the value.
+- **`splitExternalAndLocalErrors`** - (optional) Determines whether to distinguish local origin failures from external
```

Comment:
what are the types of local failures vs external errors?

Reply:
local failures:
> Various locally originated problems are detected: timeout, TCP reset, ICMP errors, etc. This detection type is supported by http router and tcp proxy.

external errors:
> server-side error responses (for HTTP traffic, 5xx responses; for TCP traffic, connection failures; for Redis, failure to respond PONG; etc.) before a consecutive total failure ejection occurs.